src/js/app.js

```diff
@@ -1,9 +1,23 @@
 import ReactDOM from "react-dom";
 import React from 'react';
-import Root from "./root";
-import { createHistory } from "history";
+import ReactFlashcards from "./components/react-flashcards";
+import thunk from "redux-thunk";
+import { Provider } from 'react-redux';
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import * as reducers from './reducers';
+
+const reducer = combineReducers(reducers);
+const finalCreateStore = applyMiddleware(thunk)(createStore);
+const store = finalCreateStore(reducer);
+
+const Root = () => {
+	return (
+		<Provider store={ store }>
+			<ReactFlashcards />
+		</Provider>
+	);
+};
 
```

Comment:
@krissalvador27 sets root component that contains the `Flashcards` component. added in a store (using Redux for state management) that will be used later on